streams/src/main/java/org/apache/kafka/streams/state/internals/CachingSessionStore.java

```diff
@@ -73,53 +99,27 @@
         return new MergedSortedCacheSessionStoreIterator<>(filteredCacheIterator, storeIterator, serdes);
     }
 
-
+    @Override
     public void remove(final Windowed<K> sessionKey) {
         validateStoreOpen();
+
```

Comment:
re `One nit: looks like this method doesn't need to be public.` Actually the test needs it to be package-private, and while on that, I feel we can either make it pure private and remove the corresponding test, or leave it public with the test if we speculate it to be used in the future.

I'd go with the second option if you do not have a strong opinion.